README.md

```diff
@@ -93,9 +93,11 @@ new Application. Fill in the values as follows:
 
 - **User**: Use the lookup (magnifying glass) to find your superuser
 - **Redirect uris**: The URL where MicroMaster’s will be running, followed by "/complete/edxorg/".
-If you're running it via Docker, run ``docker-machine ip`` from the host machine to get the
-container IP. MicroMaster’s runs on port ``8079`` by default, so this value should be something
-like ``http://192.168.99.100:8079/complete/edxorg/``
+ **Linux users:** the MicroMaster’s URL will be `localhost:8079`. **OSX users:** In a terminal window on the
+ host machine where the proper Docker environment has been set up (`eval "$(docker-machine env mm)"`), the
+ MicroMaster's IP can be found by running ``docker-machine ip`` from the host machine. MicroMaster’s runs on
```

Comment:
you need to specify the name of the machine, so it is `docker-machine ip <machine_name>`